ds701_book/20-Recommender-Systems.qmd

```diff
@@ -81,40 +79,43 @@ Some examples:
 * Priority inbox & spam filtering (Google)
 * Online dating (OK Cupid)
```

Comment:
Maybe add Tinder? I don't know if Ok Cupid is still a thing!

Reply:
still operating apparently, but changed to match.com.